### ISSUE TITLE
Feature Request: add pitch accent data

### ISSUE THREAD

**Issue by kufii** [davidsansome/tsurukame#317]
Very helpful when trying no master Japanese phonetics.

Here’s a script for the browser that adds them for an idea of what I’m talking about.

https://community.wanikani.com/t/userscript-wanikani-pitch-info/18745

**Comment by turtlemaster19** [comment]
Agree. For now, use the audio, that's why it's there!

**Comment by kufii** [comment]
> Agree. For now, use the audio, that's why it's there!

Yep agreed! Although I do find occasionally with certain words the exact pitch pattern can be hard to tell with audio alone
